docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,627 @@
+Head over to the **Filtering** menu and select a tag from the drop down list to view all labeled images.
+
+### Train a model
+
+After tagging and filtering your images, begin training your model.
+
+Click the **Train Model** button.
+Name your model.
+To match the name we used in our [bedtime songs program](#program-your-bedtime-songs-bot), name your model `"shape-classifier-model"`.
+Select **Multi label** as the model type, which accounts for multiple tags.
+
+Then select the tags that you used to label your toys and click **Train Model**.
+
+Read through our guide to [training a new model](/manage/ml/train-model/) for more information.
+
+## Use your ML Model to sing songs to your kids
+
+Now, deploy and configure your ML Model on your robot, and configure your webcam to act as a shape classifier.
+Use Viam's Go SDK to program the robot so that if your webcam "sees" (captures image data containing) a toy, the robot knows to play a particular song through its computer's speakers.
+
+### Configure your webcam to act as a shape classifier
+
+[Deploy the model](/services/ml/) to the robot and configure a [Vision Service](/services/vision/) of model `mlmodel` to use this model.
+
+To match the name we used in our [bedtime songs program](#program-your-bedtime-songs-bot), name your Vision Service `"shape-classifier"`.
+
+At this point, the full **Raw JSON** configuration of your robot should look similar to the following:
+
+{{< tabs >}}
+{{% tab name="JSON Template" %}}
+
+``` json {class="line-numbers linkable-line-numbers"}
+{
+ "packages": [
+   {
+     "name": "shapes",
+     "version": "latest",
+     "package": "20055b44-c8a7-4bc5-ad93-86900ee9735a/shapes"
+   }
+ ],
+ "services": [
+   {
+     "name": "shape-classifier-model",
+     "type": "mlmodel",
+     "model": "tflite_cpu",
+     "attributes": {
+       "model_path": "${packages.shapes}/shapes.tflite",
+       "label_path": "${packages.shapes}/labels.txt",
+       "num_threads": 1
+     }
+   },
+   {
+     "name": "shape-classifier",
+     "type": "vision",
+     "model": "mlmodel",
+     "attributes": {
+       "mlmodel_name": "shape-classifier-model"
+     }
+   }
+ ],
+ "components": [
+   {
+     "name": "cam",
+     "type": "camera",
+     "model": "webcam",
+     "attributes": {},
+     "depends_on": []
+   }
+ ]
+}
+```
+
+If you copy and paste this template into your robot's **Raw JSON** configuration, make sure to edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Record bedtime songs
+
+Now, capture the audio of the songs you want your bot to play.
+
+- Record or download the audio files you want to use to your computer in <file>.mp3</file> format.
+- Make the names of the files match the classifier tags you used: for example, <file>square.mp3</file>.
+- Place these files in the same directory on your computer as your SDK code.
```

Comment:
You're saying this before the code files are created - and then later giving detailed info about how to create that

Reply:
Oh yes, good point!